stable/influxdb/values.yaml

```diff
@@ -262,3 +262,10 @@ initScripts:
   scripts:
     init.iql: |+
       CREATE DATABASE "telegraf" WITH DURATION 30d REPLICATION 1 NAME "rp_30d"
+
+backup:
+  enabled: false
+  schedule: "0 0 * * *"
+  destination: gs://bucket/influxdb
+  google:
```

Comment:
Should we make a change to `google`? e.g gcp or gcloud?  I believe can make sense better.

Reply:
yep I went with gcs (Google Cloud Storage)